src/screens/OtherProfile/index.js

```diff
@@ -693,10 +701,10 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[800],
     fontWeight: 'bold',
     fontSize: 14,
-    color: colors.black,
+    color: COLORS.black,
     paddingBottom: 12,
     borderBottomWidth: 2,
-    borderBottomColor: colors.bondi_blue
+    borderBottomColor: COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color and border color properties have been updated to use the new `COLORS` object from the `theme` module. This is a good practice for maintaining consistency across the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->